buildpipeline/DotNet-CoreClr-Trusted-Mac.json

```diff
@@ -126,23 +140,49 @@
         "failOnStandardError": "false"
       }
     },
+    {   
+      "environment": {},
+      "enabled": true,
+      "continueOnError": true,
+      "alwaysRun": true,
+      "displayName": "Copy Files to: $(Build.StagingDirectory)\\BuildLogs",
+      "timeoutInMinutes": 0,
+      "refName": "CopyFiles1",
+      "task": {
+        "id": "5bfb729a-a7c8-4a78-a7c3-8d717bb7c13c",
+        "versionSpec": "2.*",
+        "definitionType": "task"
+      },
+      "inputs": {
+        "SourceFolder": "$(PB_GitDirectory)",
+        "Contents": "**/*.log",
+        "TargetFolder": "$(Build.StagingDirectory)\\BuildLogs",
+        "CleanTargetFolder": "false",
+        "OverWrite": "false",
+        "flattenFolders": "false"
+      }
+    },
     {
+      "environment": {},
       "enabled": true,
-      "continueOnError": false,
+      "continueOnError": true,
       "alwaysRun": true,
-      "displayName": "Copy Publish Artifact: BuildLogs",
+      "displayName": "Publish Artifact: BuildLogs",
       "timeoutInMinutes": 0,
+      "refName": "PublishBuildArtifacts1",
       "task": {
-        "id": "1d341bb0-2106-458c-8422-d00bcea6512a",
+        "id": "2ff763a7-ce83-4e1f-bc89-0ae63477cebe",
         "versionSpec": "1.*",
         "definitionType": "task"
       },
       "inputs": {
-        "CopyRoot": "",
-        "Contents": "*.log",
+        "PathtoPublish": "$(Build.StagingDirectory)\\BuildLogs",
+        "Contents": "**/*.log",
```

Comment:
hmm did you test the changes?
I don't see any `Contents` property for the task `Publish Build Artifacts`. Is it going to brake it or it just works.